src/app/(pages)/(TwoColumn)/member/delete/page.jsx

```diff
@@ -0,0 +1,114 @@
+"use client";
+
+import Breadcrumb from '@/components/common/Breadcrumb'
+import PageTitle from '@/components/common/PageTitle'
+import {
+  getLabels,
+  getMemberInfo,
+  deleteMember,
+} from "@/components/common/fetchData";
+import Menu from "@/components/common/Menu";
+import Link from "next/link";
+import { useUser } from "@/components/common/userContext";
+import { useState, useCallback, useEffect } from "react";
+import { useRouter } from "next/navigation";
+import AlertSuccess from "@/components/ui/AlertSuccess";
+
+export default function Delete() {
+  const contentDirectory = getLabels();
+  const content = contentDirectory.deleteProfile;
+  const [memberInfo, setMemberInfo] = useState([]);
+  const { user, storeUser } = useUser();
+  const router = useRouter();
+  const [alert, setAlert] = useState(false);
+
```

Comment:
I think we don't need to apply useCallback() unless no dependencies deined.
How about simply adding that function into useEffect()?
like:
```js
  useEffect(() => {
    const memberInfoFunction = async () => {
    try {
      const info = await getMemberInfo();
      setMemberInfo(info.details);
    } catch (error) {
      console.error("Error fetching member information", error);
    }
  };
  
    memberInfoFunction();
  }, []);
```

Reply:
Replaced by UseEffect()
Also applied the change in `member/edit/page.jsx` 
 https://github.com/diverta/front_next_media/pull/4/commits/aa9001e0f7c9008ccd3acb477cf43634cdde1913